src/linter/block_linter.go

```diff
@@ -720,11 +720,17 @@ func (b *blockLinter) checkFunctionCall(e *ir.FunctionCallExpr) {
 	call := resolveFunctionCall(b.walker.ctx.sc, b.classParseState(), b.walker.ctx.customTypes, e)
 	fqName := call.funcName
 
-	if call.canAnalyze {
-		b.checkCallArgs(e.Function, e.Args, call.info)
-		b.checkDeprecatedFunctionCall(e, &call)
+	if !call.isClosure {
```

Comment:
Is it enough to only check fo `!isClosure` here? It checked for canAnalyze previously as well.
Do we actually need canAnalyze now?

Reply:
This is ok, for checks from this body we do not need `canAnalyze`.